torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -1108,6 +1129,9 @@ struct GraphFuser {
         }
         any_fused = true;
         auto maybe_group = tryFuse(fused_cat, input);
+        if( !maybe_group ) {
+          continue;
```

Comment:
at this point `any_fused` is already set to `true`, even if no concat fusion happens, which will throw off subsequent checks.

Reply:
Thanks. Moved down "any_fused = true;" after all early bail-out are done.

Reply:
Please adjust `canFuseWithConcat` instead. The point here was to make sure that this function is sufficient for `tryFuse` to succeed, which is checked by the assert below.